macos/package_files/3.9.3/preinstall.sh

```diff
@@ -17,6 +17,27 @@ else
     launchctl setenv WAZUH_PKG_UPGRADE true
 fi
 
+if [ WAZUH_PKG_UPGRADE ]; then
+    mkdir /Library/Ossec/config_files/
```

Comment:
If the `config_files` directory previously exists, this line will fail. To avoid this, you can create this directory by running `mkdir -p`. In addition, don't hardcode the installation directory. There's a variable for this.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/57e1e49aa5d206859120786b6ec4357d6b9bb8ad